source/mainnet/net/guides/tokens.rst

```diff
@@ -0,0 +1,82 @@
+.. include:: ../../variables.rst
+.. _tokens:
+
+========================
+Tokens in the wallet
+========================
+
+You can add, inspect, and send tokens in the |bw|.
+
+.. dropdown:: |bw|
+
+    To access tokens and token transactions, click Tokens |token-bw|. Tabs appear for tokens.
+
+    .. dropdown:: Add tokens
+
+        #. Click **Manage**.
+
+            .. image:: ../images/browser-wallet/manage-tokens.png
+                :width: 50%
+
+        #. Enter a smart contract address on which to look for tokens. Click **Look for tokens**. If you get an error this means that the smart contract doesn’t exist, the smart contract doesn’t live up to the CIS-2 standard, or the smart contract doesn’t contain any tokens.
+
+            .. image:: ../images/browser-wallet/add-token-contract.png
+                :width: 50%
+
+        #. Select the token(s) you want to add. You can click on a token to see more information about it and then click **X** to return to the list to select tokens. You can also search by token ID at the top of the screen. Click **Update tokens** once you have made your selection(s).
+
+            .. image:: ../images/browser-wallet/select-tokens.png
+                :width: 50%
+
+        If you have purchased tokens on a marketplace, you can add the tokens to your wallet from the dApp or service so you do not have to go through the steps above. The screen below appears when the purchase is triggered and your wallet is connected. Click **Add selected** to add the selected tokens to your wallet.
```

Comment:
The issue with this is that it makes it sound like any marketplace will do this, and not a "optional" thing that they may implement, and decide themselves when to trigger. :thinking: 
```suggestion
        If you have purchased tokens on a marketplace, it can suggest tokens to be added directly without going through the steps above. In this case the screen below appears when your wallet is connected and the service suggests the tokens to be added. Click **Add selected** to add the selected tokens to your wallet.
```
Something like this?